README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
Ok, so I'm not necessarily calling you a liar, but I do believe that it is factually wrong that you need kafka up and running to do almost anything in HQ and I think including it as a startup requirement here actively harms the getting started docs since it just makes the process more painful.